src/Service/Extractor/MethodExtractor.php

```diff
@@ -18,6 +18,18 @@ public function extractMethodAnnotations(\ReflectionMethod $objectMethod, $annot
     {
         $annotations = [];
 
+        $docComment = $objectMethod->getDocComment();
+        if ($docComment
+            && false !== strpos(strtolower($docComment), '@inheritdoc')
```

Comment:
Using `strpos` for searching "@inheritdoc" is not safe, since `{@inheritdoc}` is an inline tag that can be used in the long description to inherit the parent long description inlined, eg:
```
/**
 * Makes bars
 *
 * This class generates bars using the main algorithm.
 */
class bar
{
}
 
/**
 * Makes chocolate bars
 *
 * There are two aspects to this class.
 * {@inheritdoc}  In addition, the foo class
 * makes the bars chocolate
 */
class foo extends bar
{
}
```
https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlineinheritdoc.pkg.html

Also for some particular reason, the text might exist in the doc comment, without it being an actual annotation.

Reply:
Ok, solving it with checking `inheritDoc` is too risky. I now specifically fixed it for a Doctrine proxy class.

Reply:
If you do this specifically for proxy classes, which are in essence nothing more than classes in an inheritance structure, don't you think you are doing yourself short here?

What about regular parent classes?
```
class bar
{
    /**
     * @Bos\Normalize()
     */
    public function getName(): string
    {
        return 'john wick';
    }
}
 
class foo extends bar
{
    /**
     * PHPDoc without @Bos\Normalize annotation
     */
    public function getName(): string
    {
        return 'John Wick';
    }
}
```

And what about overridden methods without a docblock?

Reply:
Thought about that, I don't want to 'just' take the parent normalize annotations. 
If someone 'overwrites' a method, they need to make sure there are new annotations on the (new) method.